curator-recipes/src/test/java/org/apache/curator/framework/recipes/nodes/TestGroupMember.java

```diff
@@ -25,11 +25,13 @@
 import org.apache.curator.retry.RetryOneTime;
 import org.apache.curator.test.BaseClassForTests;
 import org.apache.curator.test.Timing;
+import org.apache.curator.test.compatibility.CuratorTestBase;
 import org.apache.curator.utils.CloseableUtils;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 import java.util.Map;
 
+@Test(groups = CuratorTestBase.zk35TestCompatibilityGroup)
```

Comment:
Can you remind why do we need it now? It will make the test run if this env has 3.5 and up?

Reply:
We now have a (new) curator-test-zk35 module that tests ZooKeeper 3.5.x compatibility. This release will use ZooKeeper 3.6.x as the base.